Maybe required cmake should be >= 3.14

I found that `FetchContent_MakeAvailable` isn't available on cmake 3.13.0 when I tried to make this project.

```
..
-- Latest recognized Git tag is -128-NOTFOUND
-- Git HEAD is ad86920331dda3315538ab0ee30fb24727aff82d
-- Revision is 128-NOTFOUND-ad869203
-- Found LLVM: /usr/lib/llvm-6.0/include 6.0.0
-- Using static-libstdc++
-- Found LuaJIT: /usr/lib/x86_64-linux-gnu/libluajit-5.1.a;/usr/lib/x86_64-linux-gnu/libdl.so;/usr/lib/x86_64-linux-gnu/libm.so  
CMake Error at CMakeLists.txt:49 (FetchContent_MakeAvailable):
  Unknown CMake command "FetchContent_MakeAvailable".


-- Configuring incomplete, errors occurred!
See also "/home/parallels/Procmon-for-Linux/build/CMakeFiles/CMakeOutput.log".
```

That command was introduced on cmake 3.14 according to its release note.
https://cmake.org/cmake/help/v3.14/release/3.14.html#modules

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.13)
+cmake_minimum_required (VERSION 3.14)
 project (procmon)
 
 # Set the project version number.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -17,7 +17,7 @@ To contribute, fork the repository and create a branch in your fork for your wor
 * `Linux` OS (dev team is using Ubuntu 18.04)
   * Development can be done on Windows Subsystem for Linux, but Procmon cannot be executed in that environment
 * `git`
-* `cmake` >= 3.13
+* `cmake` >= 3.14
 * `libsqlite3-dev` >= 3.22
 
 ```bash

--- a/README.md
+++ b/README.md
@@ -7,7 +7,7 @@ Process Monitor (Procmon) is a Linux reimagining of the classic Procmon tool fro
 
 ## Requirements
 * OS: Ubuntu 18.04 lts with kernel >= 4.18 and kernel <= 5.3
-* `cmake` >= 3.13 (build-time only)
+* `cmake` >= 3.14 (build-time only)
 * `libsqlite3-dev` >= 3.22 (build-time only)
  
 

